doculog/main.py

```diff
@@ -19,3 +24,41 @@ def generate_changelog():
 
     if log_path.exists():
         print(f"Saved changelog to {log_path}")
+
+
+parser = ArgumentParser(
+    prog="doculog",
+    description=f"Doculog v{__version__}",
+)
+
+# parser.add_argument("-cl", "--changelog", 
+#     action="store_true",
+#     dest="cl",
+#     help="generates changelog for project"
+# )
+
+parser.add_argument("-ow", "--overwrite",
+    action="store_true",
+    dest="ow",
+    help="overwrites current CHANGELOG if present"
+)
+
+def parse():
+    args = vars(parser.parse_args())
+
+    if args["ow"]:
+        if os.path.exists("./CHANGELOG.md"):
```

Comment:
That's great, but could we instead pass this flag into generate_changelog (give the argument a default value of `False` for compatibility). That way, we can use the `log_path` variable already calculated to refer to the changelog path - this allows users to set a different name for the changelog file and it will still work. Log path is a `pathlib.Path` object so you can instead do `if log_path.exists()` and `os.remove(log_path)`